main.c

```diff
@@ -62,6 +62,21 @@ usage(void)
 	exit(1);
 }
 
+#define GETOPT_PARSENUM_WITHIN(var, min, max)				\
```

Comment:
We normally write these as
```
#define foo(bar) do { \
        blah blah blah; \
} while (0)
```

Reply:
That is to say, with the `do {` on the opening line and with a level less indentation than you have here.